### PR TITLE
docs(update): main readme(about Hovhannes)

### PR DESCRIPTION
**Pull request by Vahan11**
Please, review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,17 @@ A small project which is going to use following tech stack:
 - Google Fonts
 
 Collaborators: @vahan11, and other bootcamp students
```
Comment (vre2h) [inline README.md:10]: Plz, add Hovanes' username

File: README.md [hunk 2]
```diff
@@ -8,3 +8,17 @@ A small project which is going to use following tech stack:
...
+
+_About Hovhannes_
+
+> Hovhannes is one of our smart guys. He has a good heart.
```
Comment (hov001) [inline README.md:14]: Thank you, my smart friend

Comment (vre2h) [inline README.md:14]: Plz remove Hovanes' personal info from public project

File: README.md [hunk 3]
```diff
@@ -8,3 +8,17 @@ A small project which is going to use following tech stack:
...
+
+_About Hovhannes_
+
+> Hovhannes is one of our smart guys. He has a good heart.  
+> He is married and has a child. All in all he is a great guy,   
+> so I suggest adding his name back to our list. 
+
+P. S. 
+
+```Javascript
```
Comment (vre2h) [inline README.md:20]: I highly appreciate that you've done great work and used code style in the `readme`. Let's remove it for this PR.

**Review by vre2h — CHANGES_REQUESTED** [review]
Let's use git flow branching name convention for the next PR.

**Pull request closed without merging**